doc/user-guide.md

```diff
@@ -333,6 +335,39 @@ spec:
 
 ### Service binding
 
+#### Service Binding Operator
+
+The [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) enables application developers to bind applications together with operator-managed backing services. This can be achieved by creating a `ServiceBindingRequest` custom resource.
+
+In order for the Runtime Component Operator to pass the binding information to the application, you can specify the name of the `ServiceBindingRequest` as follows:
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: my-app
+spec:
+  applicationImage: quay.io/my-repo/my-app:1.0
+  bindings:
+    resourceRef: my-binding
+```
+
+Once the Service Binding Operator processes the `my-binding` custom resource of the type `ServiceBindingRequest`, it creates a `Secret` containing binding information provided by the backing service. The Runtime Component Operator would inject the `Secret` as environment variables into application pods.
+
+The Runtime Component Operator can also automatically detect if there is a `ServiceBindingRequest` custom resource with the same name as the application in the application namespace. If the operator detects the custom resource, it injects the binding secret into the application container.
+
+Users can turn off the auto-detection mechanism by setting the `bindings.autoDetect` parameter to `false`. The `bindings.resourceRef` parameter takes precedence over the `bindings.autoDetect` parameter. In other words, the Runtime Component Operator, injects secrets from the `my-binding` resource, even if there is a `ServiceBindingRequest` resource named `my-app`:
```

Comment:
Change "to `false`." to "to a value of `false`."
Change "even if there is a `ServiceBindingRequest` resource named `my-app`:" to "even if a `ServiceBindingRequest` resource named `my-app` exists:".